spec/features/reviews/user_can_create_review_spec.rb

```diff
@@ -19,4 +19,18 @@
     expect(page).to have_content("Very Happy")
     expect(page).to have_content("Great place to adopt a puppy! (Not great on kitties though.)")
   end
+
+  it "I see a flash message that all fields are required when missing input" do
+
+    visit "/shelters/#{@shelter_1.id}"
+
+    click_link('Add New Review')
+
+    expect(current_path).to eq("/shelters/#{@shelter_1.id}/reviews")
+
+    click_button 'Submit'
+
+    expect(current_path).to eq("/shelters/#{@shelter_1.id}/reviews")
```

Comment:
Love all the tests. I think they are good and robust. 99% is great coverage, but we can add one more to push it up to 100%